### PR TITLE
[NFC] CS: Inline a parameter in openUnboundGenericType

### PR DESCRIPTION
**Pull request by AnthonyLatsis**

### CODE REVIEW

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

File: lib/Sema/ConstraintSystem.cpp [hunk 1]
```diff
@@ -790,8 +791,7 @@ Type ConstraintSystem::openUnboundGenericType(
 
   type = type.transform([&](Type type) -> Type {
       if (auto unbound = type->getAs<UnboundGenericType>()) {
-        OpenedTypeMap replacements;
-        return openUnboundGenericType(unbound, locator, replacements);
+        return openUnboundGenericType(unbound, locator);
```
Comment (slavapestov) [inline lib/Sema/ConstraintSystem.cpp:794]: Are you sure this is NFC?

**Review by slavapestov — APPROVED** [review]
Looks good as long as the tests pass.

**Comment by slavapestov** [comment]
@swift-ci Please test source compatibility

File: lib/Sema/ConstraintSystem.cpp (hunk 1 above)
Comment (xedin) [reply lib/Sema/ConstraintSystem.cpp:794]: Based on the use it seems to be, replacements are local to a particular type...

**Review by xedin — APPROVED** [review]
LGTM!

File: lib/Sema/ConstraintSystem.cpp (hunk 1 above)
Comment (AnthonyLatsis) [reply lib/Sema/ConstraintSystem.cpp:794]: > Are you sure this is NFC?

It has to be — there is no recursion.

**Comment by AnthonyLatsis** [comment]
@slavapestov Do you want me to pursue a green Debug build?

**Comment by AnthonyLatsis** [comment]
NVM, I can see now the build was last successful almost a month ago.

**Pull request merged**